src/routes/index.svelte

```diff
@@ -73,23 +50,82 @@
     let loading = false
     let highlightedSymbolIndex = null
 
+    let isDocumentReady = false
+
     onMount(() => {
+        let DEFAULT_OPTIONS = []
+
+        if (typeof document !== 'undefined') {
+            isDocumentReady = true
+        }
+
+        selectedPriceOption = getDefaultPriceOption(price)
+
+        DEFAULT_OPTIONS = getDefaultOptions(symbols)
         fetchStockData(DEFAULT_OPTIONS)
     })
 
+    const getDefaultOptions = symbols => {
+        let symbolList
+        const storedSymbols = localStorage.getItem(LS_SYMBOLS)
+
+        if (symbols) {
+            symbolList = symbols.split(',')
+        } else if (storedSymbols) {
+            const symbolsString = storedSymbols
+            symbolList = symbolsString.split(',')
+        }
+
+        if (symbolList && symbolList.length > 0) {
+            return symbolList.map(symbol => {
+                return { symbol }
+            })
+        }
+
+        const options = [
+            {
+                symbol: 'AMZN',
+            },
+            {
+                symbol: 'GOOGL',
+            },
+            {
+                symbol: 'SHOP',
+            },
+            {
+                symbol: 'MSFT',
+            },
+        ]
+
+        return options
+    }
+
+    const getDefaultPriceOption = price => {
+        const storedPrice = localStorage.getItem(LS_PRICE)
+
+        if (price && priceOptionExists(price)) {
```

Comment:
Same here as above. Store it in local storage if it comes in as a query parameter.

Reply:
Same as previous - gets handled by the reactive declaration when selectedPriceOption is reassigned.